src/main/java/io/lettuce/core/AbstractRedisReactiveCommands.java

```diff
@@ -1521,6 +1531,16 @@ public void reset() {
         return createMono(commandBuilder::scriptFlush);
     }
 
+    @Override
+    public Mono<String> scriptFlushAsync() {
```

Comment:
Let's have a slightly different approach here. I'd suggest introducing an enum `FlushMode` (with SYNC and ASYNC constants) and introduce overloads (`scriptFlush(FlushMode)`, `flushall`, …) to not have individual methods for sync and async.

Reply:
Thanks. I will do it later.